docs/developers/developing_iree/developer_overview.md

```diff
@@ -103,9 +103,9 @@ Custom passes may also be layered on top of `iree-opt`, see
 [iree/samples/custom_modules/dialect](https://github.com/google/iree/blob/main/iree/samples/custom_modules/dialect)
 for a sample.
 
-### iree-translate
+### iree-compile
 
-`iree-translate` converts MLIR input into external formats like IREE modules. It
+`iree-compile` converts MLIR input into external formats like IREE modules. It
 is similar to
 [mlir-translate](https://github.com/llvm/llvm-project/tree/master/mlir/tools/mlir-translate),
 see "translation" in
```

Comment:
Ideally this doc would cover `iree-compile` _and_ `iree-translate` and explain the difference, rather than just find-replace. You can leave the examples under one or the other and I can take a pass through afterwards.